### PR TITLE
BOAC-3324: Surfaces student page link on admit page post-SIR

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-3324

### CODE REVIEW

File: boac/merged/admitted_student.py [hunk 1]
```diff
@@ -32,4 +32,12 @@
 
 def get_admitted_student_by_sid(sid):
     admit = data_loch.get_admitted_student_by_sid(sid)
+    if admit['current_sir'] == 'Yes':
```
Comment (johncrossman) [inline boac/merged/admitted_student.py]: @lyttam @sandeepmjay @paulkerschen - Have we decided on column-types in Nessie RDS? I was hoping for `boolean` whenever possible / appropriate. I might have missed a conversation while I was out sick.

Comment (paulkerschen) [reply boac/merged/admitted_student.py]: I believe we did collectively decide that booleans in Nessie RDS are preferable. If we want to switch BOA to match, we'd need to change the fixtures in loch.sql, and maybe code and tests elsewhere. Perhaps in a follow-up PR?

Comment (johncrossman) [reply boac/merged/admitted_student.py]: Okay, I will merge this. I have a follow up PR with cohort-related query. In that PR, I'll change BOA's expectation w.r.t  column-type and then follow up with @sandeepmjay

File: src/views/AdmitStudent.vue [hunk 2]
```diff
@@ -10,6 +10,11 @@
         tabindex="0">
         {{ admit.firstName }}&nbsp;{{ admit.lastName }}
       </h1>
+      <div v-if="admit.uid" class="pt-2 pb-3">
+        <router-link :id="`link-to-student-${admit.uid}`" :to="studentRoutePath(admit.uid, $currentUser.inDemoMode)">
+          View {{ admit.firstName }}&nbsp;{{ admit.lastName }}'s profile page
```
Comment (johncrossman) [inline src/views/AdmitStudent.vue]: @lyttam - We might want to use `v-html` on student name. See https://jira.ets.berkeley.edu/jira/browse/BOAC-2684

Comment (lyttam) [reply src/views/AdmitStudent.vue]: @johncrossman good tip. Amended.

**Comment by johncrossman** [comment]
Thanks @lyttam

**Pull request merged**